backtrader/brokers/ccxtbroker.py

```diff
@@ -71,15 +79,37 @@ def get_notification(self):
     def notify(self, order):
         self.notifs.put(order)
 
-    def getposition(self, data):
-        currency = data.symbol.split('/')[0]
-        return self.store.getposition(currency)
+    def getposition(self, data, clone=True):
+        # return self.o.getposition(data._dataname, clone=clone)
+        pos = self.positions[data._dataname]
+        if clone:
+            pos = pos.clone()
+
+        return pos
```

Comment:
This doesn't work if the you start the strategy when you are already holding a position.